packages/labs/test/tagInputTests.tsx

```diff
@@ -98,6 +98,31 @@ describe("<TagInput>", () => {
             assert.deepEqual(onAdd.args[0][0], [NEW_VALUE]);
         });
 
+        it("is invoked on blur when addOnBlur is true", done => {
+            const onAdd = sinon.stub();
+            const wrapper = mount(<TagInput values={VALUES} addOnBlur={true} onAdd={onAdd} />);
+            // simulate typing input text
+            wrapper.setProps({ inputProps: { value: NEW_VALUE } });
+            wrapper.find("input").simulate("change", { currentTarget: { value: NEW_VALUE } });
+            const fakeEvent = { flag: "yes" };
+            wrapper.simulate("blur", fakeEvent);
+            setTimeout(() => {
```

Comment:
nit: Add a comment explaining why `setTimeout` is necessary. Is this to wait for the focus to change?

Reply:
Added comment